py/code_intelligence/util.py

```diff
@@ -8,4 +11,18 @@ def write_items_to_json(output_file, results):
     for i in results:
       json.dump(i, hf)
       hf.write("\n")
-  logging.info("Wrote %s items to %s", len(results), output_file)
\ No newline at end of file
+  logging.info("Wrote %s items to %s", len(results), output_file)
+
+def parse_issue_spec(issue):
+  """Parse an issue in the form {owner}/{repo}#{number}
+
+  Args:
+    isue: An issue in the form {owner}/{repo}#{number}
+
+  Returns:
+    owner, repo, number
+  """
+  m = ISSUE_RE.match(issue)
+  if not m:
+    return None, None, None
+  return m.group(1), m.group(2), int(m.group(3))
```

Comment:
Maybe return a named tuple or a Dict?

Reply:
Sorry looks like this got merged before I saw this comment.